vars/pipeline2ATX.groovy

```diff
@@ -368,7 +372,12 @@ def createTestStep(row, debug) {
         description.addAll(crawlRows(child, true))
         child = child.nextTreeSibling
     }
-    testStep.put("description", description.join(""))
+    def allowedSchemaMaxStringLength = 117
```

Comment:
allowedSchemaMaxStringLength  are 120 ;-)
Please add a comment why you use 117

Reply:
resolved in [efd391f](https://github.com/tracetronic/jenkins-library/pull/23/commits/efd391f6ebccb315d0cfa6909cd4fbc588bb41d0)